schema/resource_test.go

```diff
@@ -133,6 +133,10 @@ var _ = Describe("Resources", func() {
 		networkRed := NewResource(networkSchema, networkRedObj)
 		Expect(networkRed.PopulateDefaults()).To(Succeed())
 		expectedConfig := map[string]interface{}{
+			"default_vlan": map[string]interface{}{
```

Comment:
nit: this map appears twice in this pr, so maybe create a `var` for it

Reply:
Unfortunately both are not exactly the same. I could extract this small part into a function, but I'm not sure it would be an improvement.